CHANGELOG.md

```diff
@@ -21,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Double-width characters in URLS only being highlit on the left half
```

Comment:
I completely forgot that this is not only the case at the end of URLs, but everywhere. Thanks for catching that and going the correct route.

This sounds good to me now, though maybe styling URLS as URLs is a little more in line with the other occurrences in the change log. But this is such a minor thing that I don't really care *that* much either way.

Reply:
Yeah, 's' is plural, not part of the acronym :smile: